x-pack/plugins/uptime/public/components/common/header/action_menu.tsx

```diff
@@ -6,70 +6,12 @@
  */
 
 import React from 'react';
-import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiToolTip } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
-import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
-import {
-  createExploratoryViewUrl,
-  HeaderMenuPortal,
-  SeriesUrl,
-} from '../../../../../observability/public';
+import { HeaderMenuPortal } from '../../../../../observability/public';
 import { AppMountParameters } from '../../../../../../../src/core/public';
-import { useUptimeSettingsContext } from '../../../contexts/uptime_settings_context';
-import { useGetUrlParams } from '../../../hooks';
+import { ActionMenuContent } from './action_menu_content';
 
-const ADD_DATA_LABEL = i18n.translate('xpack.uptime.addDataButtonLabel', {
-  defaultMessage: 'Add data',
-});
-
-const ANALYZE_DATA = i18n.translate('xpack.uptime.analyzeDataButtonLabel', {
-  defaultMessage: 'Analyze data',
-});
-
-const ANALYZE_MESSAGE = i18n.translate('xpack.uptime.analyzeDataButtonLabel.message', {
-  defaultMessage:
-    'EXPERIMENTAL - Analyze Data allows you to select and filter result data in any dimension and look for the cause or impact of performance problems.',
-});
-
-export const ActionMenu = ({ appMountParameters }: { appMountParameters: AppMountParameters }) => {
-  const kibana = useKibana();
-  const { basePath } = useUptimeSettingsContext();
-  const { dateRangeStart, dateRangeEnd } = useGetUrlParams();
-
-  const syntheticExploratoryViewLink = createExploratoryViewUrl(
-    {
-      'synthetics-series': {
-        dataType: 'synthetics',
-        time: { from: dateRangeStart, to: dateRangeEnd },
-      } as SeriesUrl,
-    },
-    basePath
-  );
-
-  return (
-    <HeaderMenuPortal setHeaderActionMenu={appMountParameters.setHeaderActionMenu}>
-      <EuiFlexGroup alignItems="flexEnd" responsive={false} style={{ paddingRight: 20 }}>
-        <EuiFlexItem>
-          <EuiToolTip position="top" content={<p>{ANALYZE_MESSAGE}</p>}>
-            <EuiButtonEmpty
-              href={syntheticExploratoryViewLink}
-              color="primary"
-              iconType="visBarVerticalStacked"
-            >
-              {ANALYZE_DATA}
-            </EuiButtonEmpty>
-          </EuiToolTip>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiButtonEmpty
-            href={kibana.services?.application?.getUrlForApp('/home#/tutorial/uptimeMonitors')}
-            color="primary"
-            iconType="indexOpen"
-          >
-            {ADD_DATA_LABEL}
-          </EuiButtonEmpty>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </HeaderMenuPortal>
-  );
-};
+export const ActionMenu = ({ appMountParameters }: { appMountParameters: AppMountParameters }) => (
+  <HeaderMenuPortal setHeaderActionMenu={appMountParameters.setHeaderActionMenu}>
+    <ActionMenuContent />
```

Comment:
There weren't any test for these button controls, so I added some while I was in here. I extracted this content to a new component because we don't have much interest in testing the `HeaderMenuPortal` API.